src/ixdat/config.py

```diff
@@ -15,9 +15,10 @@ class Config:
     """
 
     def __init__(self):
-        self.standard_metadata_suffix = "ix"
-        self.standard_data_suffix = "ixdata"
+        self.standard_metadata_suffix = ".ix"
```

Comment:
Good. Adding the dot in there, also seems compatible with (https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix)

Reply:
Yeah, was scratching my head about files ending with `..ix` (two dots).
Also it seems numpy.save cannot will always add `.npy` extension if it's not there.